src/components/TextInput/BaseTextInput.js

```diff
@@ -222,6 +235,8 @@ class BaseTextInput extends Component {
             {},
         );
         const isMultiline = this.props.multiline || this.props.autoGrowHeight;
+        const defaultInputValue = this.state.value || this.props.placeholder;
+        const defaultHiddenInputValue = this.state.hiddenInputValue || this.props.placeholder;
```

Comment:
I'm not sure why these variable name start with default.

Reply:
naming can be improved for sure

Reply:
Thanks, that is better in my opinion :)